content/articles/2017-11-15-withr-2.1.0.Rmd

```diff
@@ -0,0 +1,234 @@
+---
+title: withr 2.1.0
+date: '2017-11-15'
+slug: withr-2.1.0
+author: Jim Hester
+categories: [package]
+description: >
+  withr 2.1.0 is now available on CRAN.
+photo:
+  url: https://unsplash.com/photos/V1YEEItoOTE
+  author: Suganth
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  comment = "#>",
+  collapse = TRUE
+)
+library(withr)
+```
+
+[withr 2.1.0](http://withr.r-lib.org) is now available on CRAN!
+[withr](http://withr.r-lib.org) makes working with global state in R safer and
+less error prone. It also has no non-base dependencies so is easy to include in
+packages.
+
+Install the latest version with:
+```{r, eval = FALSE}
+install.packages("withr")
+```
+
+# Why withr?
```

Comment:
I really wish this could be "Whither withr"

Reply:
It can, even if it is not quite correct!